docs/narr.rst

```diff
@@ -98,17 +98,17 @@ object.  They cannot be changed once the message id is created:
   >>> robot.domain = "planetexpress"
   Traceback (most recent call last):
   ...
-  TypeError: readonly attribute
+  AttributeError: readonly attribute
 
   >>> robot.default = u"${name} is not a robot."
   Traceback (most recent call last):
   ...
-  TypeError: readonly attribute
+  AttributeError: readonly attribute
 
   >>> robot.mapping = {u'name': u'Bender'}
   Traceback (most recent call last):
   ...
-  TypeError: readonly attribute
+  AttributeError: readonly attribute
```

Comment:
This is strange: We raise a TypeError in `__setattr__` but an AttributeError shows up here in Python 3.
Just PyPy3 still has the `TypeError` of Python 2. – That's strange.

Raising an `AttributeError` in the codes does not break any other test. – Would this be an idea?

Reply:
`PyPy3` does not break on GHA because the Sphinx doctests are not run.